source/_docs/guides/circleci.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Continuous Integration with CircleCI
```

Comment:
Suggest a different title so users can easily differentiate this guide with the more advanced/in-depth guide from Greg. Maybe something with "essential" in the title? Same suggestion for the description below.